src/pages/en/guides/troubleshooting.md

```diff
@@ -11,16 +11,17 @@ Astro provides several different tools to help you troubleshoot and debug your c
 
 Here are some common error messages you might see in the terminal, what they might mean, and what to do about them.
 
-### Transform failed with X error
+### Transform failed with "Unexpected `export`"
```

Comment:
Just checking, is this exactly how the error message is presented? Quotation marks and code ticks? I think that's what we had before, so that the message would read the way it would be seen by the user.